### PR TITLE
[ENH] Save Data: Support saving to Excel

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Implements #3096.

##### Description of changes
Continued from #3306.
Remove support for .ods, since it didn't work.
Implement simple save to Excel, support only .xslx (xlwt didn't work with .xslx).

##### Includes
- [X] Code changes
- [X] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/data/io.py [hunk 1]
```diff
@@ -1013,6 +1023,28 @@ def read(self):
...
+                          data.domain.class_vars,
+                          data.domain.metas))
+        formatters = [cls.formatter(v) for v in vars]
+        zipped_list_data = zip(data.W if data.W.ndim > 1 else data.W[:, np.newaxis],
+                               data.X,
+                               data.Y if data.Y.ndim > 1 else data.Y[:, np.newaxis],
+                               data.metas)
+        headers = cls.header_names(data)
+        workbook = xlsxwriter.Workbook(filename)
+        print(filename)
```
Comment (janezd) [inline Orange/data/io.py]: As some would say, this is `print`.

Comment (ajdapretnar) [reply Orange/data/io.py]: 😱

File: Orange/widgets/data/tests/test_owsave.py [hunk 2]
```diff
@@ -47,7 +48,9 @@ def test_writer(self):
             self.widget.compress = d
             self.widget.filetype = t
             self.widget.update_extension()
-            self.assertEqual(len(self.widget.get_writer_selected().EXTENSIONS), 1)
+            print(self.widget.get_writer_selected())
```
Comment (janezd) [inline Orange/widgets/data/tests/test_owsave.py]: I'd like to point out that this is a `print`, too. :)

**Comment by kernc** [comment]
> Implement simple save to Excel, support only .xslx (xlwt didn't work with .xslx).

xlwt only supported the old, stable .xls format.

IIUC, the proposed change now introduces inability to _read_ files of said format.

**Pull request merged**